src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
@Nadahar If the above command didn't give the expected result, i get one more:
I just "someone to test it :P
`ffmpeg -hwaccel cuvid -c:v h264_cuvid -i jellyfish-200-mbps-4k-uhd-hevc-10bit.mkv -c:v h264_nvenc -preset superfast -profile:v main -vf "hwdownload,format=nv12,format=yuv420p,hwupload_cuda" -c:a aac -b:a 320k -ar 48000 -f ismv -movflags isml+frag_keyframe hevc_test1.mp4 -benchmark`